### PR TITLE
1 pull request mac

### PR DESCRIPTION
**Pull request by andrii-kramarenko-exa**
changed app

### CODE REVIEW

File: 12obj/App.config [hunk 1]
```diff
@@ -1,6 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
     </startup>
-</configuration>
\ No newline at end of file
+</configuration>
```
Comment (githcom) [inline 12obj/App.config:6]: please change this line

File: 12obj/App.config [hunk 2]
```diff
@@ -1,6 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
```
Comment (githcom) [inline 12obj/App.config:2]: add another initialization

**Review by githcom — CHANGES_REQUESTED** [review]
please make changes

File: 12obj/App.config (hunk 1 above)
Comment (andrii-kramarenko-exa) [reply 12obj/App.config:6]: its ok

File: 12obj/App.config [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
     </startup>
-</configuration>
```
Comment (andrii-kramarenko-exa) [inline 12obj/App.config:6]: //fdfsdfsd

File: 12obj/App.config (hunk 2 above)
Comment (githcom) [reply 12obj/App.config:2]: adde amnother way for init

File: 12obj/App.config [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
\ No newline at end of file
+</configuration>
+ Hfdksjhlkhjs 
+
+Fs fs
+
```
Comment (andrii-kramarenko-exa) [inline 12obj/App.config:10]: sdd

**Pull request merged**